db/database_test.go

```diff
@@ -28,6 +28,7 @@ func TestDatabase_OfferTable(t *testing.T) {
 	infoA := &swap.Info{
 		ID:       types.Hash{0x1},
 		Provides: coins.ProvidesXMR,
+		Status:   types.ExpectingKeys,
```

Comment:
The uninitialized value is no longer valid for serialization, so I needed to add arbitrary values to a bunch of tests.